recipes/angelscript/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conans import CMake, ConanFile, tools
+
+import os
+
+
+class AngelScriptConan(ConanFile):
+    name = "angelscript"
+    license = "zlib"
+    homepage = "http://www.angelcode.com/angelscript"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "An extremely flexible cross-platform scripting library designed to allow applications to extend their functionality through external scripts."
+    topics = "angelscript", "angelcode", "script", "scripting", "embedded"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [False, True],
+        "fPIC": [False, True],
+        "no_exceptions": [False, True],
+        "build_framework": [False, True],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "no_exceptions": False,
+        "build_framework": False,
+    }
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            headers={"User-Agent": "ConanCenter"}
+        )
+
+    def configure(self):
+        if self.settings.os == "Windows" or self.options.shared:
```

Comment:
```suggestion
        if self.settings.os == "Windows":
```

Reply:
I got a build error because I wasn't deleting the `fPIC` option on shared builds. Apparently you have to delete it: https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H007